### ISSUE TITLE
ionic resources command  - causes "Error parsing upload response"

### ISSUE THREAD

**Issue by CarlSteven** [driftyco/ionic-cli#180]
When running ionic resources, I am getting a "Error parsing upload response" when uploading my splash.png/splash.psd up to the server.

**Comment by adamdbradley** [comment]
Should be fixed now, thanks

**Comment by zapzqc** [comment]
I have the same issue。 cli :1.3.2

**Comment by rdumo01** [comment]
I have the same issue with the most recent version
![snap 2015-01-15 at 12 13 01](https://cloud.githubusercontent.com/assets/10357010/5756460/0b06a8ae-9cb0-11e4-8933-068662b3858d.png)

**Comment by deiucanta** [comment]
Same issue over here. 

![screen shot 2015-01-15 at 13 30 21](https://cloud.githubusercontent.com/assets/2116280/5756672/b97cb4d2-9cba-11e4-966e-c2deef9b185c.png)

**Comment by softBarbarian** [comment]
Same issue here.
Also see #183 .

**Comment by softBarbarian** [comment]
Here is the ionic info output:
```
OS: Mac OS X Yosemite
Node Version: v0.10.34
Cordova CLI: 4.1.2
Ionic CLI Version: 1.3.2
Xcode version: Xcode 6.1.1 Build version 6A2008a 
ios-sim version: 3.1.1 
```
Here is my terminal output:
```
Ionic icon and splash screen resources generator 
 uploading icon.png...
 uploading splash.png...
 ✗ Error parsing upload response: 
```

**Comment by CarlSteven** [comment]
Immediately after @adamdbradley 's fix it was working perfectly, but looks like it is broken for me again.
Here's my ionic info:
![image](https://cloud.githubusercontent.com/assets/7702743/5758081/933cd50a-9c87-11e4-88fe-25966967c6a7.png)

**Comment by jarhed** [comment]
Agreed, I'm having the same issue.

`
Jareds-MacBook-Pro:ionOvergrown jared$ ionic resources -i
Ionic icon resources generator 
 ✗ icon source file not found in any of these directories: resources, resources/ios
 ✗ valid icon source files: icon.psd, icon.ai, icon.png
 uploading android/icon.png...
 ✗ Error parsing upload response: 
`

**Comment by jarhed** [comment]
Confirmed, I have the same issue.

MAC 10.10
Ionic 1.3.2

**Comment by jbavari** [comment]
Thanks for reporting everyone.

We're looking into it now! We've got a hunch that its the back end service handling the requests rather than the CLI.

**Comment by jarhed** [comment]
Working again nicely, thanks!

**Comment by rdumo01** [comment]
same for me ! Nice job, thanks

**Comment by gauchini** [comment]
Same error for me
Ionic icon and splash screen resources generator 
 uploading icon.png...
 uploading splash.png...
 ✗ Image server temporarily unavailable: <!DOCTYPE html>
    <html>
    <head>
      <meta name="viewport" content="width=device-width, initial-scale=1">
      <style type="text/css">
        html, body, iframe { margin: 0; padding: 0; height: 100%; }
        iframe { display: block; width: 100%; border: none; }
      </style>
    <title>Application Error</title>
    </head>
    <body>
      <iframe src="//s3.amazonaws.com/heroku_pages/error.html">
        <p>Application Error</p>
      </iframe>
    </body>
    </html>

**Comment by adam-back** [comment]
I was able to create icons after two tries with the `--icon` flag, but splash keeps failing. I updgraded node and the ionic-cli. Followed sizing instructions from [ionic blog].(http://blog.ionic.io/automating-icons-and-splash-screens/)

ionic-cli: 1.6.1
cordova: 5.1.1
node: v0.12.7
npm: 2.11.3

Splash resource is named `splash.png`, is 2.3 MB, and 2208x2208.
![screen shot 2015-07-10 at 3 58 19 pm](https://cloud.githubusercontent.com/assets/8163408/8630578/8b0bcfba-271c-11e5-9d1d-985add5b27b0.png)
